.vib/logstash/goss/logstash.yaml

```diff
@@ -0,0 +1,11 @@
+file:
+  /opt/bitnami/logstash/config/logstash.yml:
+    exists: true
+    filetype: file
+    contains:
+      - "path.data: /bitnami/logstash/data"
+  /opt/bitnami/logstash/config/log4j2.properties:
+    exists: true
+    filetype: file
+    contains:
+      - "Console"
```

Comment:
I would add a `uid/gui` check for the created `logstash/logstash` at the postunpack stage.

https://github.com/bitnami/containers/blob/34ae65e2f29bc7f97111aa8895af62d04b16c068/bitnami/logstash/7/debian-11/rootfs/opt/bitnami/scripts/logstash/postunpack.sh#L18

Reply:
Good catch! I've added it now.